libcloudforensics/gcp.py

```diff
@@ -288,11 +416,9 @@ def GetDisk(self, disk_name):
       raise RuntimeError(error_msg)
     return disk
 
-  def CreateDiskFromSnapshot(self,
-                             snapshot,
-                             disk_name=None,
-                             disk_name_prefix='',
-                             disk_type='pd-standard'):
+  def CreateDiskFromSnapshot(
```

Comment:
I think this (and next occurrences) was initially a response to https://github.com/google/cloud-forensics-utils/pull/28#discussion_r412227417

It's fine for now; I opened #79 to keep track of this.